lib/rubygems/util/spdx.rb

```diff
@@ -0,0 +1,303 @@
+module Gem
+  class SPDX
+    IDENTIFIERS = %w(
```

Comment:
I'm not sure what freezing buys.  It's a constant, which should indicate that the value supposed to be... :star2: constant :star2:

If someone really really wants to modify the constant, they can use `const_set` and there's nothing we can do about it.